main.py

```diff
@@ -68,14 +68,17 @@ def change_read():
 
 def delete_book():
 
-    book_id = ui.get_book_id()
-    book = store.get_book_by_id(book_id)
-    answer = ui.ask_question("Type DELETE to confirm you'd like to delete this book: ")
-    if answer == "DELETE":
-        Book.delete(book)
-    else:
-        ui.message("Returning to main menu. Book has not been deleted.")
-        main()
+    try:
+        book_id = ui.get_book_id()
+        book = store.get_book_by_id(book_id)
+        answer = ui.ask_question("Type DELETE to confirm you'd like to delete this book: ")
+        if answer == "DELETE":
+            Book.delete(book)
+        else:
+            ui.message("Returning to main menu. Book has not been deleted.")
+            main()
```

Comment:
@whagestad Code is running good on my end. However, I believe you don't need to add line 79 main(). The try and except returns the menu already after unsuccessful book deletion.